engine/engine.go

```diff
@@ -14,6 +15,13 @@ import (
 	logging "github.com/ipfs/go-log/v2"
 )
 
+const (
+	defaultWorkers = 1
```

Comment:
join this with the `1` constant in the flag?

Reply:
Removed this, flag already has default.